src/Rendering/Bs4FormRenderer.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/**
+ * This file is part of the Nextras community extensions of Nette Framework
+ *
+ * @license    New BSD License
```

Comment:
Please remove this line, this repo is MIT and actually the Bs3 probably should be MIT too and definitely new code :)